digdag-core/src/main/java/io/digdag/core/schedule/ScheduleExecutor.java

```diff
@@ -199,8 +199,14 @@ private void runSchedule(ScheduleControl lockedSched)
 
             Config scheduleConfig = SchedulerManager.getScheduleConfig(def);
             boolean skipOnOvertime = scheduleConfig.get("skip_on_overtime", boolean.class, false);
+            Optional<Integer> skipDelay = scheduleConfig.getOptional("skip_delayed_by", Integer.class);
```

Comment:
https://github.com/treasure-data/digdag/pull/752#discussion_r181556017

https://github.com/treasure-data/digdag/blob/1ed050a5a3c0708d85cde7c37dfef06e2094f4e7/digdag-server/src/main/java/io/digdag/server/WorkflowExecutionTimeoutEnforcer.java#L79

With DurationParam.class not Integer.class, user can use a time unit like `30s` / `1m` as they want. If a specified figure is handled as only seconds, the option name needs to have `second` or something.

Could you try to use DurationParam.class?

Reply:
Thank you for your advice.